src/styles/styles.css

```diff
@@ -31,4 +31,36 @@ main {
     background-size: cover;
     background-position: center;
     background-repeat: no-repeat;
+}
+
+.main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+section {
+    width: 100%;
+    display: block;
+}
+
+.promo {
+    width: 90%;
+    margin: 40px auto;
+}
+
+.promo__text {
+    border-radius: 8px;
+    padding: 20px 30px;
+    color: white;
+    font-weight: 400;
+    font-size: 1.4rem;
+    text-align: center;
+    background-color: var(--Blue);
+    box-shadow: 0px 8px 4px 0px rgba(0, 0, 0, 0.2);
```

Comment:
let's see if we can add this shadow to variables, so we can reuse it in other styles